app/src/main/java/com/example/javaopencommerce/order/OrderFactory.java

```diff
@@ -0,0 +1,46 @@
+package com.example.javaopencommerce.order;
+
+import static java.util.stream.Collectors.toUnmodifiableList;
+
+import com.example.javaopencommerce.Amount;
+import com.example.javaopencommerce.Value;
+import com.example.javaopencommerce.Vat;
+import com.example.javaopencommerce.item.dtos.ProductDto;
+import com.example.javaopencommerce.order.OrderDetails.SimpleProduct;
+import com.example.javaopencommerce.order.dtos.OrderDetailsDto;
+import java.time.LocalDate;
+import java.util.List;
+
+class OrderFactory {
+
+  OrderDetails toOrderModel(OrderDetailsDto orderDetailsDto) {
+    List<SimpleProduct> orderBody = getOrderBody(orderDetailsDto.getCard().getProducts());
+
+    return OrderDetails.builder()
+        .id(orderDetailsDto.getId())
+        .paymentMethod(PaymentMethod.valueOf(orderDetailsDto.getPaymentMethod()))
+        .orderStatus(OrderStatus.NEW)
+        .paymentStatus(PaymentStatus.BEFORE_PAYMENT)
```

Comment:
Does incoming DTO have those fields? Those will always be new/before_payment when we'll want just use OrderFactory as a mapper...